test/tests/multiapps/picard/picard_master.i

```diff
@@ -76,6 +76,7 @@
     app_type = MooseTestApp
     positions = '0 0 0'
     input_files = picard_sub.i
+    clone_master_mesh = true
```

Comment:
OK - so what are the scenarios in the sub app now? I assume we have a "file" or "generated" mesh there. We are going to push the mesh down from the master though. Can that block be omitted? Perhaps we should have a test both ways? One with and one without the mesh? You might counter that it doesn't make sense to remove the mesh from the sub in the case where you want to run the sub file by itself?

Just trying to make sure we don't introduce a bunch of new edges cases with this quick hack.

Reply:
The `Mesh`, `MeshModifiers`, and `MeshGenerators` will all be ignored (see the changes in the actions in this PR) and there is a line print-out from multiapp to tell users that master mesh is cloned and will be used by subapps. Note that this parameter is only valid in `MultiApps/*` input syntax. At that time, users should know what they are trying to do. The input for a multiapp should be always able to run stand-alone (I think we should enforce this rule). I believe this PR is safe in sense of no introduction of new edge cases.